setup.py

```diff
@@ -156,6 +156,13 @@ def has_ext_modules(self):
             os.path.relpath(os.path.join(rootname, filename), datapath))
         print("setup.py - copy {} to {}".format(src, dst))
         shutil.copyfile(src, dst)
+      for filename in fnmatch.filter(filenames, "*.py"):
+        src = os.path.join(rootname, filename)
+        dst = os.path.join(
+            rootpath,
+            os.path.relpath(os.path.join(rootname, filename), datapath))
+        print("setup.py - copy {} to {}".format(src, dst))
+        shutil.copyfile(src, dst)
```

Comment:
Just wondering why adding this now, was it grabbing some incorrect files?

Also could you just reuse the similar loop above and just nest under a list of extensions? like

```
for file_pattern in ["*.so", "*py"]:
  for filename in fnmatch.filter(filenames, file_pattern):
   ...
```

Reply:
Thanks @BryanCutler. The PR has been updated. Previously, we pick up py files in the source tree, and pick up .so files in bazel-bin. However, for gRPC, it consists of generated code in python (.py) based on endpoint.proto file. Those files are generated as part of the build so they are placed in bazel-bin (by bazel). So setup.py has to be updated to capture the py files in bazel-bin as well.